### ISSUE TITLE
Weird flickering when viewing graphs in standard mode on smartphone

### ISSUE THREAD

**Issue by fedarko** [marbl/MetagenomeScope#159]
This is at mgsc.umiacs.io, so it's an older version of the viewer interface than the one contained in this repository.

For some reason, the screen started flickering between black and white rapidly when I zoom in on graphs on my phone (observed this with the E. coli and sample Bandage graphs, as well as the SRS049959 graph). I noticed nodes not being "drawn," also. This is a serious issue, both because it makes interacting with the graph difficult and because the flickering could negatively affect people with photosensitivity.

Guessing this is a Cytoscape.js bug that has since been resolved -- I will try to reproduce this with the latest version of MetagenomeScope after the refactoring I've been doing. For now I'm going to temporarily take down the demo so that this issue doesn't affect people.

For reference, using Google Chrome 77.0.3865.73 on Android.

**Comment by fedarko** [comment]
Looks like updating the Cytoscape.js version didn't do anything to fix the problem. This is super weird. (It works fine on my laptop.)

I've updated mgsc.umiacs.io to show a warning explaining the situation. i'll examine this in more detail tomorrow -- this may just be an artifact of an old version of the viewer interface being used, but i have no idea why i'd only just start seeing this error now. another explanation is this is a recent chrome bug, so i can try replicating with other smartphone browsers.

**Comment by fedarko** [comment]
Something even more bizarre: the SPQR mode stuff (for the marygold example graph) seems to work fine on my phone. But the standard mode stuff (even for the marygold graph, which is super tiny) still exhibits the problem.

My guess, then, is this has to do with some sort of setting I'm using in just the standard mode but not the SPQR mode. This is corroborated by the fact that I can't reproduce this error in any of the Cytoscape.js demos.

maybe updating the xdot2cy.js -- etc versions might fix this? doubtful, but worth a shot.

**Comment by fedarko** [comment]
For what it's worth, I can't replicate this issue any more. My guess is that it was just a temporary phone browser problem with the canvas operations used by Cytoscape.js. Just restored the demo to normal.
